Makefile

```diff
@@ -190,125 +192,112 @@ SPCC_DEPS := check_python_format pylint_src pylint_tests mypy mypy_test pydocsty
 # To repeat a single test, apply something like @pytest.mark.repeat(3) on the test function
-# -n Const because most tests include parallel execution using all CPUs, too many 
-# parallel tests would lead to contention. Thus Const is set to something much lower than num_cpus
-# --durations=10 is to show the 10 slowest tests
-# --capture=tee-sys is to make sure that, in case of crash, we can search for "Forcing seed to" in 
-# stdout in order to be able to reproduce the failed test using that seed
-# --global-coverage-infos-json=global-coverage-infos.json is to dump the coverage report in the file 
-# --cov PATH is the directory PATH to consider for coverage
+# -svv disables capturing of stdout/stderr and enables verbose output
+# -n N is to set the number of CPUs to use for pytest. We set it to N_CPU=4 by default because most 
+# tests include parallel execution using all CPUs and too many parallel tests would lead to 
+# contention. Thus N is set to something lower than the overall number of CPUs
+# --count N is to repeate all tests N times (with different seeds). Default is to COUNT=1.
 # --randomly-dont-reorganize is to prevent Pytest from shuffling the tests' order
-# --cov-report=term-missing:skip-covered is used to print the missing lines for coverage withtout 
-# taking into account skiped tests
-# --count N is to repeate all tests N times (with different seeds). Default is to 1.
 # --randomly-dont-reset-seed is to make sure that, if we run the same test several times (with
 # @pytest.mark.repeat(3)), different seeds are used, even if things are still deterministic using 
 # the main seed
-# --cache-clear is to clear all Pytest's cache at before running the tests. This is done in order to prevent inconsistencies 
-# when running pytest several times in a row (for example, in the CI)
-.PHONY: pytest # Run pytest on all tests
-pytest:
+# --capture=tee-sys is to make sure that, in case of crash, we can search for "Forcing seed to" in 
+# stdout in order to be able to reproduce the failed test using that seed
+# --cache-clear is to clear all Pytest's cache at before running the tests. This is done in order to 
+# prevent inconsistencies when running pytest several times in a row (for example, in the CI)
+.PHONY: pytest_internal # Run pytest
+pytest_internal:
 	poetry run pytest --version
-	poetry run pytest tests/ \
+	poetry run pytest $(TEST) \
 	-svv \
-	-n 4 \
-	--durations=10 \
-	--capture=tee-sys \
-	--global-coverage-infos-json=global-coverage-infos.json \
-	--cov=$(SRC_DIR) \
-	--cov-fail-under=100 \
-	--randomly-dont-reorganize \
-	--cov-report=term-missing:skip-covered \
+	-n $(N_CPU) \
 	--count=$(COUNT) \
+	--randomly-dont-reorganize \
 	--randomly-dont-reset-seed \
+	--capture=tee-sys \
 	--cache-clear \
 	${PYTEST_OPTIONS}
 
-.PHONY: pytest_one # Run pytest on a single file or directory (TEST) a certain number of times (COUNT)
-pytest_one:
-	poetry run pytest \
-	-svv \
-	-n $$(./script/make_utils/ncpus.sh) \
```

Comment:
So we completely remove the command?

Reply:
no it's still there, I just moved it below in the file

Reply:
Oh alright got it.